src/mmrelay/matrix/events.py

```
@@ -0,0 +1,738 @@
+            try:
+                handler_result = plugin.handle_room_message(room, event, text)
+                if inspect.isawaitable(handler_result):
+                    found_matching_plugin = await handler_result
+                else:
+                    found_matching_plugin = bool(handler_result)
+
+                if found_matching_plugin:
+                    facade.logger.info(
+                        f"Processed command with plugin: {plugin.plugin_name} from {event.sender}"
+                    )
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)